packages/core/services/AnnotationService/DatabaseAnnotationService/index.ts

```diff
@@ -140,4 +140,13 @@ export default class DatabaseAnnotationService implements AnnotationService {
             }, [] as string[])
             .filter((annotation) => !annotationSet.has(annotation));
     }
+
+    public async createAnnotation(annotation: Annotation): Promise<any> {
+        const tableName = this.dataSourceNames.sort().join(", ");
+        return await this.databaseService.addNewColumn(
```

Comment:
Can remove `await` and `async` from this.

Reply:
removed in [73a8316](https://github.com/AllenInstitute/biofile-finder/pull/406/commits/73a83161edfb1645a108729511b49fe509ca46e7)